.buildkite/bk.integration.pipeline.yml

```diff
@@ -35,14 +35,14 @@ steps:
   - group: "Stateful IT (Sudo): Ubuntu"
     key: integration-tests
     depends_on:
-      - package-it
+      # - package-it
       - integration-ess
     steps:
       - label: "Default"
         key: stateful-ubuntu-default-sudo
         command: |
-          buildkite-agent artifact download build/distributions/** . --step 'package-it'
-          .buildkite/scripts/steps/integration_tests_tf_sudo.sh "default" "^(TestAPMConfig|TestDiagnosticsOptionalValues|TestIsolatedUnitsDiagnosticsOptionalValues|TestDiagnosticsCommand|TestIsolatedUnitsDiagnosticsCommand|TestEventLogFile|TestFakeComponent|TestFakeIsolatedUnitsComponent|TestOtelFileProcessing|TestOtelLogsIngestion|TestOtelAPMIngestion|TestPackageVersion)$$"
+          buildkite-agent artifact download build/distributions/** . --step 'package-it' --build "01928ad3-fc0a-45de-81d5-0bfb1fdc0caa"
```

Comment:
```suggestion
          buildkite-agent artifact download build/distributions/** . --step 'package-it' --build "0192dd5c-9edc-456c-9ba4-8b9078997b6d"
```

Reply:
@pazone 
I applied the suggestion to be able to test this for the time being...
If everything checks out I would like to have the changes to the pipeline in a separate PR after we merge this one with only the changes to the integration test framework

Reply:
Started a [new test build](https://buildkite.com/elastic/elastic-agent-extended-testing-bk/builds/26) for that. Let's make sure that it works and then we can revert the pipeline changes